app/controllers/admin/talks_controller.rb

```diff
@@ -33,7 +33,7 @@ def talk
     def talks
       @talks ||= search_against(Talk)
         .includes(:tags, :event, :speaker, :group)
-        .page(params[:page])
+        .page(params[:page]).sorted_by_date
```

Comment:
If I will remove `.sorted_by_date` scope from here - no tests will show me that this is incorrect.
Please add feature spec [here](https://github.com/pivorakmeetup/pivorak-web-app/blob/development/spec/features/admin/talks/read_spec.rb#L1) and check the talks ordering.